modules/gdscript/gdscript_parser.h

```diff
@@ -344,6 +344,7 @@ class GDScriptParser {
 			OP_LOGIC_OR,
 			OP_TYPE_TEST,
 			OP_CONTENT_TEST,
+			OP_NOT_CONTENT_TEST,
```

Comment:
I feel like this name is odd (it is not a content test, so what it is?). I would prefer something like `OP_NEGATIVE_CONTENT_TEST` or maybe `OP_ABSENCE_TEST`.

Reply:
With the previous comment regarding not-adding-a-new-variant-op, this won't be needed anymore.
But just in case, we do decide to keep the Variant op: I like OP_NEGATIVE_CONTENT_TEST, more descriptive.